rubin_scheduler/scheduler/surveys/field_survey.py

```diff
@@ -198,9 +199,9 @@ def _generate_survey_name(self, target_name=None):
         else:
             self.survey_name = f"Field {self.ra_deg :.2f} {self.dec_deg :.2f}"
 
-    @cached_property
+    @property
```

Comment:
For *regular* field surveys, where the RA/Dec of the healpix desired doesn't change, this should be a cached_property. 
For *altaz* field surveys, it should *not* be cached, and just a @property.